Source/JavaScriptCore/API/PASReportCrashPrivate.cpp

```diff
@@ -25,6 +25,9 @@
 
 #include "config.h"
 #include "PASReportCrashPrivate.h"
+#include <wtf/Compiler.h>
+
+WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
```

Comment:
Why aren't you using our compiler-specific pragmas?

Reply:
Good point. It looks like this is the first Apple-port-specific annotation in the codebase. To make regex easy I'll use "Darwin ports" (since "non-Apple" ports is taken, so a regex for "Apple" would be complicated).